### PR TITLE
Add modifier for a vertical oblique Equator in -JO

### LINKED ISSUE #2786: Oblique Mercator Projection - Orientation of the Oblique Equator
The Feature Request is that GMT includes the option to plot oblique projections with the oblique equator parallel to the y-axis.

The oblique equator in an Oblique Mercator projection is plotted parallel to the x axis.    It would be useful to be able to plot the oblique equator parallel to the y-axis.   

**Are you willing to help implement and maintain this feature?** No. Regrettably I do not have sufficient coding skills.

### PR DESCRIPTION
**Pull request by PaulWessel**
**Description of proposed changes**

Since GMT was created, the oblique Mercator projection maps all had one thing in common: The oblique Equator was mapped to be the positive _x_-axis.  This worked well, but for folks who wanted to make oblique maps aligned with a feature whose orientation is more north-south than east-west this layout is inconvenient.  For instance, an oblique Mercator map aligned with the elongation of California should have that oblique Equator become the vertical y-axis instead.  Many users have requested such a feature, most recently in #2786.  Thanks to the coronavirus I found a few hours to look into this.  The end result is a new modifier **+v** to the **-JO**|**o** option that selects vertical rather than horizontal alignment for the oblique Equator.  As an example, here is a new test script for the oblique Mercator documentation:

GMT_obl_baja

I have added new tests to ensure this works well both for the northern and southern hemisphere, and that it works with the **-R**...**+u**_unit_ selection for region.  All tests pass, and in the process I fixed some related issues that led to some PostScript original updates. Closes #2786.

### CODE REVIEW

File: doc/rst/source/cookbook/map-projections.rst [hunk 1]
```diff
@@ -735,10 +735,13 @@ degrees about the projection center rather than the usual geographic
...
 parallels and meridians are not very suitable as map boundaries.
 
+Normally, the oblique Equator becomes the horizontal (*x*) axis in the projected units.
+You can select the vertical (*y*) axis by appending **+v**.
+
 When working with oblique projections such as here, it is often much more convenient
 to specify the map domain in the projected coordinates relative to the map center.
 The figure below shows two views of New Zealand using the oblique Mercator projection
-that in both cases specifies the region using **-Rk**\ -1000/1000/-500/500.  The leading
+that in both cases specifies the region using **-R**\ -1000/1000/-500/500**+uk**.  The
```
Comment (seisman) [inline doc/rst/source/cookbook/map-projections.rst]: Remember to update the old **-Rk** syntax in `doc/scripts/GMT_obl_nz.sh`.

**Comment by PaulWessel** [comment]
@seisman, is the checks stuck again?

**Comment by seisman** [comment]
Yes, don't know what's happening.

**Comment by seisman** [comment]
The checks work well on the master branch. So I think it's OK to merge it and check the CI reports there.

**Pull request merged**